misc/distributed_search/speclib_config.py

```diff
@@ -0,0 +1,39 @@
+# modify config to include fasta for library prediction
```

Comment:
could this be named like an action, e.g. "create_config_for_library_prediction.py` ?

Reply:
As with the renaming in AlphaDIA, and considering that these python scripts are not user-exposed, I would be in favor of merging a stable version and handling renaming in distributed_search & AlphaDIA itself in a future PR